src/playground/Widgets/List/index.tsx

```diff
@@ -0,0 +1,63 @@
+import { ProgressBar } from "@/components/Charts/ProgressChart"
+
+interface BaseListItem {
+  name: string
+}
+
+interface DefaultListItem extends BaseListItem {
+  value: number | string
+}
+
+interface ProgressListItem extends BaseListItem {
+  value: number
+}
+
+type ListItem = DefaultListItem | ProgressListItem
+
+interface ListProps {
+  items: ListItem[]
+  title?: string
+  variant?: "default" | "progress"
+}
+
+const ListItem = ({
+  item,
+  variant = "default",
+}: {
+  item: ListItem
+  variant?: "default" | "progress"
+}) => (
+  <div className="mb-2 flex w-full items-center last:mb-0">
+    <span className="flex-grow truncate text-muted-foreground">
+      {item.name}
+    </span>
+    {variant === "default" ? (
```

Comment:
According to the suggestion above, you won't need to check the variant here but `typeof item.value` instead